### ISSUE TITLE
Mermaid diagrams are rendering zoomed in so that only a small portion is visible

### ISSUE THREAD

**Issue by awwx** [Roam-Research/issues#501]
**Describe the bug**

Mermaid diagrams are rendering zoomed in, so that only a small portion of the diagram is visible.

```
* {{mermaid}}
  * flowchart TD
  * A --> B
  * B --> C
```

<img width="585" alt="Screen Shot 2020-08-22 at 8 47 40 AM" src="https://user-images.githubusercontent.com/143754/90956390-2ce7dc80-e454-11ea-99ec-223193526373.png">

**System Information:**
 - Roam version: 0.7.3-e272efd5
 - Device: MacBook 
 - OS: macOS Mojave 10.14.6
 - Browser: Chrome 84.0.4147.135 (Official Build) (64-bit)

**Comment by awwx** [comment]
This appears to now be fixed, though the client is still on 0.7.3-e272efd5.

**Comment by stasparshin** [comment]
v. 0.7.3-aba1b017 - not fixed for me :(

**Comment by awwx** [comment]
@stasparshin odd, I'm on 0.7.3-aba1b017 now also, but it's still working for me 🤷‍♂️

Have you tried force reloading the page?

Is there anything different in your environment (browser, OS)?

**Comment by charleshan** [comment]
Happening for me as well. 

System Information:
* Roam version: 0.7.3-02c189c8
* Device: MacBook
* OS: macOS Mojave 10.14.6
* Browser: Chrome 85.0.4183.83 (Official Build) (64-bit)

**Comment by awwx** [comment]
@charleshan  Yes, I'm also running 0.7.3-02c189c8, Chrome 85.0.4183.83 (Official Build) (64-bit), MacBook, macOS Mojave 10.14.6... and I'm seeing it now too.

**Comment by marcelchastain** [comment]
In chrome devtools I was able to temporarily work around this by adjusting the following css properties, listed in order found in the DOM:
* `.react-resizable { height: auto; }`
* `.rm-mermaid svg { min-width: unset; }`

YMMV. I modified the the DOM node of a single mermaid diagram and it worked fine, but you might need more precise css selectors if you want to apply a global override via a stylesheet.
Also, this gets my basic diagrams to show up but it might not produce the exact display/rendering behavior intended by the Roam team.

Grateful to the team for supporting mermaid syntax, and I hope this helps while the larger issue is being investigated 👍

**Comment by peteraritchie** [comment]
> In chrome devtools I was able to temporarily work around this by adjusting the following css properties, listed in order found in the DOM:
> 
> * `.react-resizable { height: auto; }`
> * `.rm-mermaid svg { min-width: unset; }`
> 
> YMMV. I modified the the DOM node of a single mermaid diagram and it worked fine, but you might need more precise css selectors if you want to apply a global override via a stylesheet.
> Also, this gets my basic diagrams to show up but it might not produce the exact display/rendering behavior intended by the Roam team.
> 

Thanks! The `.rm-mermaid svg { min-width: unset; }` was enough for it to work for me.
